imports/server/garbage-collection.ts

```diff
@@ -45,8 +44,15 @@ async function cleanup() {
   await Servers.removeAsync({ _id: { $in: deadServers } });
 }
 
+export async function markServerDead(id: string) {
```

Comment:
nit: I'd call this `cleanupDeadServer` instead of `markServerDead` since we're not really doing mark/sweep but are cleaning up after this particular dead server.